webapp/pages/code-of-conduct.vue

```diff
@@ -25,7 +25,7 @@
 
       <p>
         {{ $t('code-of-conduct.get-help') }}
-        <a class="hc-editor-link-blot" href="moderation@human-connection.org" target="_blank">
+        <a class="hc-editor-link-blot" href="mailto:moderation@human-connection.org" target="_blank">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-22T11:52:25Z" title="Saturday, February 22nd 2020, 12:52:25 pm +01:00">Feb 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -25,7 +25,7 @@
 
       <p>
         {{ $t('code-of-conduct.get-help') }}
-        <a class="hc-editor-link-blot" href="moderation@human-connection.org" target="_blank">
+        <a class="hc-editor-link-blot" href="mailto:moderation@human-connection.org" target="_blank">
```


Hey @Ruben80 it looks as if this branch introduces the same changes as #3133. Probably you wanted to commit/push something else?